README.md

```diff
@@ -186,6 +186,11 @@ The following is not supported, although it has some use cases; see [Issue #18](
 
 * optional property assignment: `a?.b = c`
 
+The following are not supported, as it does not make much sense, at least in practice; see [Issue #4 (comment)](https://github.com/tc39/proposal-optional-chaining/issues/4#issuecomment-373446728):
+
+* optional super: `super?.()`, `super?.foo`
+* anything that resemble to property access or function call, but is not: `new?.target`, `import?.('foo')`, etc.
```

Comment:
these are called "metaproperties" fwiw

Reply:
ah true, `import()` is not. `new.target`, `function.sent`, `import.meta` are metaproperties.

Reply:
Maybe working along the lines of:

---

> There are a variety of syntactic elements that optional chaining does not support, as there is no compelling use-case and/or the behavior would be nonsensical:

* optional super: `super?.()`, `super?.foo`, in a non-subclassing class.
* `new.target`, or `import.meta`, which are not in fact property access, etc.

---

I think, if I'm reading the spec-text correctly, we actually support `super?.foo` if we happen to be in a class that's subclassing something?

Reply:
@bcoe `super?.foo` would require `super` to be a PrimaryExpression. (We do support `super.foo?.()` and `super.foo()?.bar`, but these are just normal call and access cases.)

Reply:
"optional metaproperty access or `import` call" might work as wording here.

Reply:
I understand, [I was reading this](http://www.ecma-international.org/ecma-262/10.0/index.html#prod-SuperProperty), which I suppose implies this syntax would work:

```
super['foo']?.bar
super.foo?.bar
super('foo')?.bar
```

my bad 👍

Reply:
Right, just a suggestion given @ljharb's comment. I'm fine with any of the above (though if we go with the existing text we should correct `resemble to` to `resembles`).